src/js/components/posts-page.component.js

```diff
@@ -26,6 +28,26 @@ export default class PostsPage {
                 const modalTemplate = userComponent.template;
                 this.body.prepend(userComponent.createNode(modalTemplate));
             })
+            const commentShowBtn = node.querySelector("#showCommentsBtn");
+            commentShowBtn.addEventListener("click", async () => {
```

Comment:
Just one small thing here, maybe it would be better to store comments on post object as array of Comment objects. That way we can optimize the performance and reduce the number of network calls. Currently, when we click button, it shows loader and makes a network call to get comments. Ideally, on the first click, we get comments and store them in Post itself and on consecutive clicks we just check if comments array is already there on Post

https://user-images.githubusercontent.com/56446932/152112144-2b281c64-76f0-4966-9abb-8f1a0d57ad6b.mov